exes/mqtt-broadcaster/inc/config/spark_plug_b_config.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <string>
+#include <string_view>
+
+#include <glaze/core/common.hpp>
+#include <tfc/utils/json_schema.hpp>
+
+namespace tfc::mqtt {
```

Comment:
instead of injecting config in struct names you can just make a subnamespace 
`tfc::mqtt::config { }`